### ISSUE TITLE
Is there any option to disable early stop of optimizer during registration

### ISSUE THREAD

**Issue by guyj93** [ANTsX/ANTsPy#58]
I find that the registration result is not so stable that in some case the registration stop very fast and the results are not good enough.

I have tried to tune 'reg_iterations' and 'grad_step' parameters in 'ants.registration' function but it didn't help.

Is there any option in 'antspy' or 'ants' to disable the early stopping of optimizer during registration?

Any help would be much appreciated !

**Comment by stnava** [comment]
you can make a pull request that exposes the convergence criterion ( see ants documentation ).

however, I suspect that this is not your real issue.

**Comment by guyj93** [comment]
Thanks!

I modified my code to implement this feature and I found that it can solve problem in several samples.

In fact, I am using ants to do image registration between CT and MR images of pelvis part (CT is the moving image), and I want to fix the bone parts. I have the bone segmentation results and I don't want to do non-rigit registration within the bone parts, so I first do rigid registration and then fill the bone parts with background color in both MR and CT before SyN, to force the SyN method align the bone parts. Then, the unstable results maybe happen in the SyN results. 

Is there a better way to fix part of image when using SyN? I found that adding mask won't help to fix the region without the mask.
